packages/nodes-base/nodes/ExecuteWorkflow/methods/resourceMapping.ts

```diff
@@ -0,0 +1,24 @@
+import type {
+	ILoadOptionsFunctions,
+	ResourceMapperField,
+	ResourceMapperFields,
+} from 'n8n-workflow';
+
+export async function getWorkflowInputs(
+	this: ILoadOptionsFunctions,
+): Promise<ResourceMapperFields> {
+	// TODO: take the columns from the workflow input
```

Comment:
If you need anything specified to get these from my end do let me know. Presumably you'd "just" need to get the node and call 

```ts
thatNode.getNodeParameter(`${FIELDS}.${VALUES}`, itemIndex, []) as Array<{name: string; type: FieldType;}>;
```

Or will we need a different mechanism?

Reply:
That is exactly what we will do, thanks @CharlieKolb

Reply:
We need to completely redesign the method used to load selected workflow parameters.